plugins/tech-radar/src/plugin.ts

```diff
@@ -36,6 +39,13 @@ export const techRadarPlugin = createPlugin({
   apis: [createApiFactory(techRadarApiRef, new SampleTechRadarApi())],
 });
 
+/**
+ * Main Tech Radar Page
+ *
+ * @remarks
+ *
+ * Uses {@link TechRadarPageProps} as props
+ */
 export const TechRadarPage = techRadarPlugin.provide(
```

Comment:
Yeah extensions is definitely an issue. Let's not block this PR on fixing that as it could take some time to find a nice general solution to it. I'll have a look tho